doc/TILESET.md

```diff
@@ -114,6 +123,221 @@ You can add `"rotates": true` to allow sprites to be rotated by the game automat
 
 `"multitile": true` signifies that there is an `additional_tiles` object (redundant? [probably](https://github.com/CleverRaven/Cataclysm-DDA/issues/46253)) with one or more objects that define sprites for game entities associated with this tile, such as broken versions of an item, or wall connections.  Each object in the array has an `id` field, as above, and an `fg` field, which can be a single [root name](#root-name), an array of root names, or an array of objects as above. `"rotates": true` is implied with it and can be omitted.
 
+#### Connecting terrain and furniture - `connects_to`
+
+For terrain or furniture that is intended to auto-connect using multitiles, set the property `connects_to` in the object definition (not in the tileset!) to an appropriate group:
+
+```json
+{
+    "type": "terrain",
```

Comment:
This should probably go into <https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/JSON_INFO.md#terrain> since it's not a tileset property

Reply:
Thanks! Will add it there, but would prefer to leave a a bit of information also in the tileset docs.

Reply:
Done.